plugins/mcreator-localization/help/fr_FR/rangeditem/bullet_damage.md

```diff
@@ -1 +1 @@
-Higher the number the stronger the damage the bullet deals to the entity hit.
\ No newline at end of file
+Plus le nombre est élevé, plus les dégâts infligés par la balle à l'entité touchée sont plus fortes.
```

Comment:
It's "[...] plus les dégâts infligés par la balle à l'entité touchée sont plus forts."